oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/ProcessTopologyBuilder.java

```diff
@@ -69,6 +69,8 @@ public ProcessTopologyBuilder(ModuleManager moduleManager, StorageModels storage
 
     ProcessTopology build(List<Call.CallDetail> clientCalls,
                           List<Call.CallDetail> serverCalls) throws Exception {
+        log.info("building process topology, total found client calls: {}, total found server calls: {}",
```

Comment:
```suggestion
        log.debug("building process topology, total found client calls: {}, total found server calls: {}",
```

I think this should be `debug`, rather than `infor`?